sebex/analysis/model.py

```diff
@@ -53,7 +53,8 @@ class AnalysisEntry:
 
     @property
     def is_published(self) -> bool:
-        return bool(self.releases)
+        # always publish test packages
+        return bool(self.releases) or 'sebex_test' in self.package
```

Comment:
this magic constant looks like something that should be rather configured via manifest file instead of being hardcoded 🤔 

maybe consider adding a field for project in manifest, smth like `publish: true` to let Sebex do first publish?

Reply:
Such a field field already exists in release.yaml:
![image](https://user-images.githubusercontent.com/29464251/135609144-7829cb37-915b-44bb-b683-81e6d00e847f.png)
release.yaml is generated after running `sebex release plan`. Setting `publish: true` manually works as intended. Should I delete the hardcoding and just mention this possibility in the README or still add this configuration to the manifest file?

Reply:
I thought more about adding it to manifest, not release file